data/ecosystem.yaml

```
@@ -1,142 +1,10 @@
-collections:
-  name: Ansible collections
-  description: Ansible collections offer distributions of playbooks, roles, modules, and plugins.
+ecosystem:
+  name: Ansible ecosystem
+  description: Expand your automation to a virtually unlimited set of use cases with projects managed by Red Hat.
```

```suggestion
  description: Expand your automation to a virtually unlimited set of use cases with open source projects managed by Red Hat. The projects welcome community contributions!
```
I would suggest having something saying this explicitly to avoid community contributors coming only to the awesome ansible projects:) Feel free to reformulate it if needed and you agree

---

Thanks, @Andersson007 that's a good suggestion.

---

@Andersson007 just tweaked your suggestion PTAL

---

@oraNod now it's much better, thanks! LGTM